src/site/_includes/components/Aside.js

```diff
@@ -22,6 +25,33 @@ function Aside(content, type = 'note') {
   const locale = getLocaleFromPath(this.page && this.page.filePathStem);
 
   let prefix;
+
+  // Renders the new aside
+  if (isDesignSystemContext(this.page.filePathStem)) {
```

Comment:
If I have the page, can't I just read its data here to find this out? (i.e., what else is available on `this`? 11ty is bad at types, time to `log`). I feel like the collection backing `isDesignSystemContext` is adding a weird layer of abstraction we don't need.

Reply:
Nah 11ty only gives [page variables](https://www.11ty.dev/docs/data-eleventy-supplied/#page-variable-contents). It's a weird layer: you're right. It just helps us get over the line then it'll get yoinked out again